languoids/tree/indo1319/clas1257/germ1287/nort3152/west2793/high1289/high1286/midd1349/mode1258/baye1239/bava1246/cent1967/vien1238/md.ini

```diff
@@ -9,6 +9,6 @@ macroareas =
 countries = 
 	Austria (AT)
 links = 
+	[German (Viennese)](https://wals.info/languoid/lect/wals_code_gvi)
```

Comment:
See above.

Reply:
The division of Central Bavarian into Upper Austrian and Viennese is a bit random though. The main division of Central Bavarian dialects is into Western Central and Eastern Central Bavarian with the boundary running north-south between Passau and Linz. (Source: https://commons.wikimedia.org/wiki/File:Bairischer_Sprachraum2.PNG). Then we could place the city dialect Viennese into Eastern Central Bavarian, but it's a bit hard to assign Upper Austrian, as this is situated right at the boundary between the Western and Eastern dialects.

So we could do something like that:

Bavarian
- Central Bavarian
-- Western Central Bavarian
--- Munich German
-- Eastern Central Bavarian
--- Viennese German

Reply:
The dialect inventories in Glottolog are often not really complete or systematic. Often these inventories are driven by "need", i.e. by what varieties linguists write about or do research with. So, the fact that Upper Austrian and Viennese are included in WALS is good reason to assign glottocodes to these varieties - which means including them as "dialects".

Regarding the Western/Eastern split: I'd leave this to @d97hah to decide. Renaming "Viennese German" makes sense, I think.

Reply:
On the other hand it is simply common to use the term "Viennese German" disregarding the linguistic affiliation.